src/expr/src/scalar/func.rs

```diff
@@ -2878,6 +2935,15 @@ impl fmt::Display for BinaryFunc {
             BinaryFunc::RangeContainsElem { rev, .. } => {
                 f.write_str(if *rev { "<@" } else { "@>" })
             }
+            BinaryFunc::RangeContainsRange { rev, .. } => {
+                f.write_str(if *rev { "<@" } else { "@>" })
+            }
+            BinaryFunc::RangeOverlaps => f.write_str("&&"),
+            BinaryFunc::RangeAfter => f.write_str(">>"),
+            BinaryFunc::RangeBefore => f.write_str("<<"),
+            BinaryFunc::RangeOverleft => f.write_str("&<"),
+            BinaryFunc::RangeOverright => f.write_str("&>"),
+            BinaryFunc::RangeAdjacent => f.write_str("-|-"),
```

Comment:
is this an operator we parse?

Reply:
Yep https://github.com/MaterializeInc/materialize/pull/17341/files#diff-628ea3914035abe87302037eb9284f9a6478f190dfb57ac0f54310c92e14f51fR837